### ISSUE TITLE
Add support for ProxyJump

### ISSUE THREAD

**Issue by pikeas** [moul/advanced-ssh-config#202]
Breaking out from #190.

**Comment by robinbowes** [comment]
I'd like to see this too.

**Comment by ihipop** [comment]
Any Update on This issue?

**Comment by danielhoherd** [comment]
I think ProxyJump is the most significant missing feature that is preventing me from adopting `assh` as a default replacement for `ssh`. I think a decent middle ground would be if `assh config build` would put ProxyJump configs into place when they are found in assh-config.yaml files, because then I could still manage my ssh configs via assh. As it is now, I would have to maintain vanilla ssh configs where ProxyJump is required along with assh configs, and be careful not to overwrite those configs when using assh.

**Comment by ihipop** [comment]
since  year 2017. Any Update on This issue?

**Comment by milkpirate** [comment]
2021, anything new?

**Comment by milkpirate** [comment]
Soon 2022... this is the only reason why we dont use assh. We really would want to have this! Please stop us from using Jinja templates :(

**Comment by sergei-dyshel** [comment]
@milkpirate 
Not to defend author's position on this issue, but just curios - where does requirement  of `ProxyJump` come from? I'm not expert in OpenSSH implementation but I was always sure that `ProxyJump <bastion>` is just syntactic sugar for `ProxyCommand="ssh -W %h:%p <bastion>"`  and that's exactly what `assh` does, together with alias resolving, nested proxies etc.

So having in `assh.yml`:
```
my-host:
	ProxyJump: bastion
```
is pretty synonymous to
```
my-host:
	Gateways: [bastion]
```
I understand that we can avoid an additional invocation of `assh`  with `ProxyJump` , but how can this be a showstopper?

**Comment by milkpirate** [comment]
Yeah, we have a wild setup, especially this "alias resolving, nested proxies etc." is necessary...

**Comment by sergei-dyshel** [comment]
But what `ProxyJump` gives you that `assh` can't do?
